lib/testing/FakeVis.hpp

```diff
@@ -58,9 +58,6 @@
  *                      Default is False.
  * @conf  frequencies   Array of UInt32. Definition of frequency IDs for
  *                      mode 'test_pattern_freq'.
- * @conf  dataset_id    Int. Use a fixed dataset ID and don't register
- *                      states. If not set, the dataset manager will create
- *                      the dataset ID.
  * @conf  sleep_before  Float. Sleep for this number of seconds before
```

Comment:
Why is this removed? The option still exists.

Reply:
I thought it didn't exist because I did a search for it, but it just had a different name. I've added it back in.